### DISCUSSION TITLE
Classroom: Error Accepting Assignments (Help).

### DISCUSSION THREAD

**Discussion by DrAdamLee** [community/community#135904] [GitHub Education]
### Hi everyone,

We have a course, and are the instructors. Our students are suddenly unable to accept our assignments, with universal reports of the error: **This assignment can no longer be accepted. Please contact your teacher. They might need to edit or recreate the assignment.**

When trying to investigate and edit the assignment in question, we get most options greyed out - for example, starter code, is greyed out.

Upon changing the already-not-out-of-date deadline, students were able to accept the assignment, but then had repositories that were completely empty. The starter code in settings shows no repository linked, and is greyed out, and non-clickable. 

Does anyone know if we are doing something wrong? We tried to create new assignments, to test if these were working: and we get assignments accepted with the same error, above. The starter code is the same as we have always used, and is unchanged. We have several other lessons on offer with different starter code, but get the same results.

Any thoughts would be hugely appreciated - our course is fully locked down at the moment, and we have no idea what to do.

Many thanks, in advance, for your time and help!

**Comment by sabaferdous12** [comment]
Having same issues. Trying to create assignments in different classrooms but on acceptance stage having the error: 

`This assignment can no longer be accepted. Please contact your teacher. They might need to edit or recreate the assignment.`

**Comment by ADWolfe-LoyNO** [comment]
> ### Hi everyone,
> We have a course, and are the instructors. Our students are suddenly unable to accept our assignments, with universal reports of the error: **This assignment can no longer be accepted. Please contact your teacher. They might need to edit or recreate the assignment.**
> 
> When trying to investigate and edit the assignment in question, we get most options greyed out - for example, starter code, is greyed out.
> 
> Upon changing the already-not-out-of-date deadline, students were able to accept the assignment, but then had repositories that were completely empty. The starter code in settings shows no repository linked, and is greyed out, and non-clickable.
> 
> Does anyone know if we are doing something wrong? We tried to create new assignments, to test if these were working: and we get assignments accepted with the same error, above. The starter code is the same as we have always used, and is unchanged. We have several other lessons on offer with different starter code, but get the same results.
> 
> Any thoughts would be hugely appreciated - our course is fully locked down at the moment, and we have no idea what to do.
> 
> Many thanks, in advance, for your time and help!

has this been fixed?

**Comment by nisxant69** [comment]
The problem you’re facing, where students cannot accept assignments and the message “This assignment can no longer be accepted” appears, combined with greyed-out starter code and missing repository links, suggests there might be an issue with how the assignments or starter code repositories are linked or permissions set. Since changing the deadline temporarily allows students to accept assignments but results in empty repositories, it points to a disconnect between the assignment configuration and the starter code source. It’s possible that the linked starter code repository has been moved, deleted, or had its permissions changed, causing GitHub Classroom to lose access. Also, if the assignment was created with a repository that no longer exists or isn’t accessible, that would explain the greyed-out options when you try to edit it. Creating new assignments with the same starter code and seeing the same error suggests a broader issue, possibly related to repository permissions or ownership. I recommend verifying that the starter code repositories still exist, you have admin access to them, and that the GitHub Classroom app or integration has the necessary permissions to access these repos. If the repositories are in an organization, double-check organization policies or repo visibility settings. You might also want to try unlinking and relinking the starter code repository in the assignment settings or recreating the starter code repo if possible. If none of this works, reaching out to GitHub Classroom support with detailed info will be necessary since the issue could be on their platform side.

**Reply by ADWolfe-LoyNO** [reply]
How do I "reach out to GitHub Support"??

Every time I look for this I get pointed here—a freaking bulletin board. I have no idea whether anyone from GitHub is here at all. Is there an email address or a web form? Please tell me what I have missed.

**Comment by PalominoVidalCarlos** [comment]
Have an issue to reuse group assignments de assignment is copy Without Trouble but appears inactive when I edit the assignment to config apropiadamente de Page 500 erro appears I tried everything but I can't solve this issue
